pytorch/inference/docker/1.5.0/py3/Dockerfile.cpu

```diff
@@ -99,7 +100,15 @@ RUN pip install \
     pillow==7.1.0 \
     "awscli<2" \
     # cryptography required by pyopenssl and paramiko, <3.3.2 has safety vulnerabilities
-    "cryptography>=3.3.2"
+    "cryptography>=3.3.2" \
+    "paramiko==2.7.2" \
+    "Flask==1.1.2" \
+    "boto3==1.17.34" \
+    "click==7.1.2" \
+    "gevent==21.1.2" \
+    "gunicorn==20.0.4" \
+    "inotify-simple==1.3.5" \
+    "protobuf==3.15.6"
```

Comment:
Why not use `protobuf==3.11.3` like this: https://paste.amazon.com/show/chumbalk/1616446911 ? also for other imports

Reply:
Is there a hard dependency on the package versions? we generally tend to install the latest package version available on pypi and conda.